oauth/authorization.md

```diff
@@ -1,92 +1,231 @@
-- Example
-
-
-### Guided Practice
-
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
-
-### Independent Practice
-
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
-
-### Challenge
-
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
-
-
-### Check for Understanding
-
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
-
-### Supplemental Materials
-- [example website](https://example.com) - Write a very short description of how to use this optional resource.
-- [Other example website](https://otherexample.com) - Write a very short description of how to use this optional resource.
+1. [These lesson slides follow the Guided Practice section below.](). 	
+1. [Follow this video describing how to add Auth0 to an app](). 	
+### Common Mistakes / Misconceptions	
+-**"It's okay if I post my API keys to my private GitHub repo."** 
+Never push API keys to GitHub, even if you are certain the repository is secure. You might end up changing the permissions or adding a collaborator in the future. Keeping your keys off GitHub protects them from being exposed, even accidentally.	
+
+-**Logging in to an app with OAuth gives the developer access to my Google password**. 
+Passwords are never shared and never even pass through the primary app's servers. Instead, providing your credentials to Google (or another social media platform) along with information from the primary app tells Google it's okay to send a *different* piece of shared secret information--tokens--to the primary app. The tokens are now associated with specific users, and *this* is the currency used between the primary app and the third-party service.
+
+### Guided Practice	
+#### Step 1 - Configure Node.js to use Auth0
+##### Create the .env file	
+Create the .env file in the root of your app and add your Auth0 variables and values to it.
+	# .env	
+```	
+AUTH0_CLIENT_ID=YOUR_CLIENT_ID	
+AUTH0_DOMAIN=YOUR_AUTH0_DOMAIN	
+AUTH0_CLIENT_SECRET=YOUR_CLIENT_SECRET
+```
+	
+##### Install Dependencies
+To get started, install the following dependencies.	
+*`passport` - an authentication middleware for Node.js
+*`passport-auth0` - an Auth0 authentication strategy for Passport
+*`express-session` - a middleware to manage sessions
+*`dotenv` - a module to load environment variables from a .env file	
+ # installation with npm
+	
+``` npm install passport passport-auth0 express-session dotenv --save ```
+
+ ##### Configure express-session	
```

Comment:
this header doesn't work because you have a space before it.

Reply:
@alodahl It looks good to me in the preview...